ci/minimum_supported_agent_version_test.go

```diff
@@ -0,0 +1,104 @@
+//go:build ci
+
+package ci
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/ops-agent/integration_test/metadata"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	added    = flag.String("added", "", "Added files")
+	modified = flag.String("modified", "", "Modified files")
+	labels   = flag.String("labels", "", "Pull request labels")
```

Comment:
Hmmm. are we using the labels to tell who the author is? The pull request trigger by post release script doesn't attach any labels at this current moment

Reply:
We will change the release script to add the appropriate labels. This approach allows a maintainer to fix up `minimum_supported_agent_version` in case of an error, which would be very cumbersome if we checked the author instead.

Reply:
Right, good thought!